### ISSUE TITLE
small routes problem

### ISSUE THREAD

**Issue by egorodessa** [plataformatec/devise#4800]
## Precheck

- Do not use the issues tracker for help or support, try Stack Overflow.
- For bugs, do a quick search and make sure the bug has not yet been reported
- Finally, be nice and have fun!

## Environment

- Ruby **[2.5.0]**
- Rails **[5.1.5]**
- Devise **[4.4.1]**

## Current behavior

when file **routes.rb** have it:

```ruby
  devise_scope :user do
    root to: "devise/sessions#new"
  end
```

and controller generated for manual edit, then it did not reach that(generated) controller when you try to access root route.

## Expected behavior
when we follow root route it have to go through the code we put to devise generated controller session#new

**Comment by Bennet11** [comment]
Hi There, I'm new in contributing to OSS projects and would love to try and work on this issue as my first Open Source contribution.

**Comment by tegon** [comment]
Hello @egorodessa, thanks for your report.

I haven't understood exactly what's the problem. Can you provide us a simple application that reproduces this issue in isolation, or maybe list the steps to reproduce it?
This can help us to figure out what's going on.

Thanks.

**Comment by tegon** [comment]
@Bennet11 thanks for your interest in contributing to devise!
I'm not sure what's the error in this issue, so I think it's better to clarify what's need to be done first.
But if you want, I can point another issue to you. What do you think?

Thank you!

**Comment by Bennet11** [comment]
@tegon I was about to ask the same question you've asked since Im not sure what's the issue as well. Sounds about great to me. Thanks for accepting me btw ill try my best to contribute to devise.

**Comment by tegon** [comment]
@Bennet11 we have some issues with the [`Needs PR`](https://github.com/plataformatec/devise/issues?q=is%3Aopen+is%3Aissue+label%3A%22Needs+PR%22) label that are available to work on.
Do you want to work on any of those? You can chose one and if you have any questions I'd be happy to help.

**Comment by tegon** [comment]
I'm closing this issue because it has not had recent activity.
If you're still facing this on the latest version, please open a new one with all the information requested in the template.

Thank you!
